PushNotifications.md

```diff
@@ -62,7 +62,7 @@ To get push notifications setup in your own app, read [Setting up your own app](
 - Open your iOS app in Xcode: when in your project directory, run `xed ios` or double click `ios/Runner.xcworkspace` in `your_project_name/ios`
     - Ensure your application bundle ID is registered on App Store connect. This is done automatically by Xcode when you select your team under `Signing & Capabilities`.
     - Create a `.p12` certificate and upload it to the Ably dashboard to allow Ably to authenticate with APNs on behalf of you, using [How do I obtain the APNs certificates needed for iOS Push Notifications?](https://knowledge.ably.com/how-do-i-obtain-the-apns-certificates-needed-for-ios-push-notifications).
-       - When running your application debug mode, it will always use the sandbox/ development APNs environment. When running in release mode, it will always use the production environment. This is an iOS limitation. Keep this in mind when checking the `Use APNS sandbox environment?` checkbox in Ably dashboard (notifications tab). When running an application in debug mode, ensure this checkbox is ticked, and when running a release build, ensure you are using an application with this checkbox disabled. 
+       - When running your application via Xcode or your machine (Android Studio, command line), your application runs in either debug, profile or release mode. In all cases, your application will use the sandbox/ development APNs environment. When distributing your app in the App Store, Ad Hoc or through App Store Connect, it will always use the production environment. If distributing through `Development`, the sandbox/ development APNs environment is used. Keep in mind your distribution method when checking the `Use APNS sandbox environment?` checkbox in a Ably application on the Ably dashboard (notifications tab). Ensure your application connects to an Ably application configured to use production APNs if the application is being distributed publicly.
```

Comment:
Please include a space either side of `/` in sentences. Also, 'Ably application' needs to be preceded by 'an', not 'a'.

```suggestion
       - When running your application via Xcode or your machine (Android Studio, command line), your application runs in either debug, profile or release mode. In all cases, your application will use the sandbox/ development APNs environment. When distributing your app in the App Store, Ad Hoc or through App Store Connect, it will always use the production environment. If distributing through `Development`, the sandbox / development APNs environment is used. Keep in mind your distribution method when checking the `Use APNS sandbox environment?` checkbox in an Ably application on the Ably dashboard (notifications tab). Ensure your application connects to an Ably application configured to use production APNs if the application is being distributed publicly.
```